packages/shared/src/components/search/SearchBarInput.tsx

```diff
@@ -1,38 +1,6 @@
-import React, {
-  FormEvent,
-  ForwardedRef,
-  forwardRef,
-  InputHTMLAttributes,
-  MouseEvent,
-  ReactElement,
-  useContext,
-  useEffect,
-  useRef,
-  useState,
-} from 'react';
-import classNames from 'classnames';
-import {
-  RaisedLabel,
-  RaisedLabelContainer,
-  RaisedLabelType,
-} from '../cards/RaisedLabel';
-import { BaseField, FieldInput } from '../fields/common';
-import { AiIcon, MiniCloseIcon as CloseIcon } from '../icons';
-import { IconSize } from '../Icon';
-import { getFieldFontColor } from '../fields/BaseFieldContainer';
-import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
-import { useInputField } from '../../hooks/useInputField';
-import { SearchProgressBar } from './SearchProgressBar';
-import { SearchChunk, SearchProviderEnum } from '../../graphql/search';
-import { useViewSize, ViewSize } from '../../hooks';
-import { useAuthContext } from '../../contexts/AuthContext';
-import { SearchSubmitButton } from './SearchSubmitButton';
-import { MobileSearch } from './MobileSearch';
+import { FormEvent, InputHTMLAttributes } from 'react';
```

Comment:
I think we can put the remaining contents of this file somewhere else as it doesn't have any components anymore.